app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -302,6 +302,26 @@ class ProfileEditActivityTest {
     }
   }
 
+  @Test
+  fun testProfileEdit_startWithUserProfile_clickDelete_configChange_checkIfDialogRemains() {
```

Comment:
As per wiki (https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing#naming-convention) the outcome of the test is not written at the end because `checkIfDialogRemains` just means that it is checking but either the dialog is visible or not that result is not mentioned.

It should be something like:

`testProfileEdit_startWithUserProfile_clickDelete_configChange_dialogIsStillVisible`

Reply:
Ok